### ISSUE TITLE
Are these hyperparameters the final version?

### ISSUE THREAD

**Issue by darlingfirebox** [muhanzhang/pytorch_DGCNN#7]
Are these hyperparameters the final version? I have tried many times, and the classification accuracy of some datasets is quite different from that described in the paper.

**Comment by muhanzhang** [comment]
The results in the paper were generated using the Torch version. This PyTorch version by default will run batch optimization. Batch optimization is good for some datasets, and might be bad for others. You can try to disable batch optimization by setting _bsize=1_ in the "run_DGCNN.sh" file.

**Comment by darlingfirebox** [comment]
Thank you for your patience!
